scan.go

```diff
@@ -118,6 +122,8 @@ func newScanCommand() *cobra.Command {
 		"Analyze transitive dependencies")
 	cmd.Flags().IntVarP(&transitiveDepth, "transitive-depth", "", 2,
 		"Analyze transitive dependencies till depth")
+	cmd.Flags().BoolVarP(&dependencyUsageEvidence, "dependency-usage-evidence", "", false, "Enable dependency usage evidence during scan")
+	cmd.Flags().StringVarP(&codeAnalysisDbPath, "code-db", "", "", "Path to code analysis database generated by 'vet code scan'")
```

Comment:
I am thinking if we should just call this arg `--code`. Providing a `--code /path/to/code.db` is like explicitly providing & enabling code analysis based on `code.db` during scan.